### PR TITLE
ovsianko hw11-beta

### PR DESCRIPTION
**Pull request by kevinlobster**

### CODE REVIEW

File: homeworks/Ovsianko/hw-11/breakpoints.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+@media only screen and (min-width: 769px) {
```
Comment (valsydev) [inline homeworks/Ovsianko/hw-11/breakpoints.css:1]: Code style

File: homeworks/Ovsianko/hw-11/breakpoints.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    .container-clients {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-auto-columns: 1fr;
+    gap: 0px;
+    grid-auto-flow: row;
+    grid-template-areas:
+        "head-clients head-clients head-clients head-clients"
+        "cli1 cli2 cli3 cli4"
```
[the remark this answers is not in the document]
Comment (kevinlobster) [reply homeworks/Ovsianko/hw-11/breakpoints.css:62]: Client

File: homeworks/Ovsianko/hw-11/pages/cases/cases.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<html>
```
Comment (valsydev) [inline homeworks/Ovsianko/hw-11/pages/cases/cases.html:1]: good

File: homeworks/Ovsianko/hw-11/pages/publications/publications.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    text-decoration: none;
+    display: inline-block;
+}
+
+button:hover{background-color: var(--active-hover);}
+
+button:active{background-color: var(--active-active);}
+
+.p-desktop-pub{
+    display: none;
```
Comment (valsydev) [inline homeworks/Ovsianko/hw-11/pages/publications/publications.css:82]: When does `display` have non-none value?

File: homeworks/Ovsianko/hw-11/pages/topbar/topbar.css [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+        top: 70px;
+        gap: 0px;
+        flex-direction: column;
+        width: 100%;
+        text-align: center;
+        transition: 16px 0.3s;
+    }
+
+    .nav-item {
+        margin: 5.5px 0;
```
Comment (valsydev) [inline homeworks/Ovsianko/hw-11/pages/topbar/topbar.css:134]: 5.5? 🤔

**Pull request closed without merging**